data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,305 @@
+      <timeout config:type="integer">10</timeout>
+      <trusted_grub>false</trusted_grub>
+      <xen_append>splash=silent quiet console=ttyS1,115200</xen_append>
+      % if ($check_var->('VERSION', '15')) {
+      <terminal>serial</terminal>
+      <xen_kernel_append>dom0_max_vcpus=4 dom0_mem=8192M,max:8192M vga=gfx-1024x768x16 loglvl=all guest_loglvl=all</xen_kernel_append>
+      % } else {
+      <terminal>console serial</terminal>
+      <xen_kernel_append>dom0_max_vcpus=4 vga=gfx-1024x768x16 loglvl=all guest_loglvl=all</xen_kernel_append>
+      % }
+    </global>
+    <loader_type>grub2</loader_type>
+  </bootloader>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage>
+      <start_multipath config:type="boolean">false</start_multipath>
+    </storage>
+  </general>
+  <kdump>
+    <add_crash_kernel config:type="boolean">true</add_crash_kernel>
+    <crash_kernel>201M,high</crash_kernel>
+    <crash_xen_kernel>201M\&lt;4G</crash_xen_kernel>
+  </kdump>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+<interfaces config:type="list">
+  <interface>
+    <device>br0</device>
+    <bootproto>dhcp</bootproto>
+    <bridge>yes</bridge>
+    <bridge_forwarddelay>0</bridge_forwarddelay>
+    <bridge_ports>eth0 eth1</bridge_ports>
+    <bridge_stp>off</bridge_stp>
+    <startmode>auto</startmode>
+  </interface>
+  <interface>
+    <device>eth0</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+  <interface>
+    <device>eth1</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+</interfaces>
+  </networking>
+  <ntp-client>
+    <ntp_policy>auto</ntp_policy>
+    <ntp_servers config:type="list">
+      <ntp_server>
+        <address>europe.pool.ntp.org</address>
+        <iburst config:type="boolean">true</iburst>
+        <offline config:type="boolean">false</offline>
+      </ntp_server>
+    </ntp_servers>
+    <ntp_sync>systemd</ntp_sync>
+  </ntp-client>
+  <partitioning config:type="list">
+    <drive>
+      <!--device>/dev/sda</device-->
```

Comment:
Why is it commented off? Without it, which device is the partition created on?

Reply:
autoyast can detect and find out the right device. AMD machines does not use /dev/sda. it's specified it will break AMD installation.

Reply:
ok